src/validetta.js

```diff
@@ -274,21 +274,21 @@
       // Handle submit event
       $(this.form).submit(function(event) {
         // fields to be controlled transferred to global variable
-        FIELDS = event.currentTarget.getElementsByTagName('input');
+        FIELDS = this.querySelectorAll('.form-input');
```

Comment:
is this still scoped to the form that's being submitted? We don't want invalid inputs elsewhere on the page causing validation to fail